rosdep/python.yaml

```
@@ -6104,6 +6104,10 @@ python3-boto3:
     '*': ['python%{python3_pkgversion}-boto3']
     '7': null
   ubuntu: [python3-boto3]
+python3-breathe:
+  debian: [python3-breathe]
+  fedora: [python3-breathe]
```

```suggestion
  fedora: [python3-breathe]
  opensuse: [python3-breathe]
  rhel: [python3-breathe]
```

Add rhel and opensuse suggested by linter